timeless/views.py

```diff
@@ -281,7 +280,11 @@ class FakeQuery:
         def get(self, object_id):
             """Fake response on get method."""
             if object_id == 5:
-                return {"Found the object"}, HTTPStatus.OK
+                response = {
```

Comment:
@salgadobreno @emilianodellacasa I think we should get rid of these fake objects. at least move them from prod code to `test/` path. I believe it should be isolated in one test file where we are testing generic views. wdyt?

BTW the name of class is `FakeModel`, but inside `.get` method we return json object and HTTP status code like it's a view :) But it's not view. This fake object is just an attempt to simulate the logic of this method: https://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.get
So it should return None in case if object is not found or model instance in another case. do we need one more fake class `FakeModelInstance`, instance of which should be returned? :)

I see that you and code owner didn't get that the method returns incorrect value, so it means, we have to find clearer way to test generic views.

Reply:
@vryazanov @emilianodellacasa I think you are correct and I was a bit weary of the design here but since I don't know much about the frameworks and patterns in place I've tried to follow along with what was laid out before in test, mock and structure. We'd have to define some boundary though otherwise there'd be a big refactor inside this issue, also one that I'm limited to do. What I'd think more interesting for the project was if @vryazanov would review the puzzles here because there are a lot of them in this file and each seems to be describing a different design route... I mean that's just my general feeling, but back to the point I think there should either be a new bug report trying to get this sorted or the considerations could be added in a well defined puzzle for future change.

Reply:
@salgadobreno @vryazanov I agree that those fake objects should be moved to test path, and we should discuss if we really need them (maybe add this consideration to the puzzle)